src/XFShimmerLayout/Controls/ShimmerLayout.cs

```diff
@@ -106,6 +106,56 @@ public int Angle
             set => SetValue(AngleProperty, value);
         }
 
+        public CornerRadius CornerRadiusOverlayDefault
+        {
+            get => (CornerRadius)GetValue(CornerRadiusOverlayDefaultProperty);
+            set => SetValue(CornerRadiusOverlayDefaultProperty, value);
+        }
+
+        public static readonly BindableProperty CornerRadiusOverlayDefaultProperty = BindableProperty.Create(
+            nameof(CornerRadiusOverlayDefault), typeof(CornerRadius), typeof(ShimmerLayout), default(CornerRadius));
+
+
+        public Thickness PaddingOverlayDefault
+        {
+            get => (Thickness)GetValue(PaddingOverlayDefaultProperty);
+            set => SetValue(PaddingOverlayDefaultProperty, value);
+        }
+
+        public static readonly BindableProperty PaddingOverlayDefaultProperty = BindableProperty.Create(
+            nameof(PaddingOverlayDefault), typeof(Thickness), typeof(ShimmerLayout), default(Thickness));
+
+
+        #endregion
+
+
+        #region Attached Properties
+
+        public static BindableProperty CornerRadiusOverlayProperty = BindableProperty
+            .CreateAttached("CornerRadiusOverlay", typeof(CornerRadius), typeof(ShimmerLayout), default(CornerRadius));
```

Comment:
Please fix the format like on line 126:
property = BindableProperty.CreateAttached(
    nameof(.....), ... );